source_code/main_mcu/src/fido2/fido2.h

```diff
@@ -0,0 +1,24 @@
+enum ErrorCode
+{
+    SUCCESS = 0,
```

Comment:
context with the define? this sure will create some conflicts later on the development process.

Reply:
absolutely